packages/react-sdk/src/components/box.ws.ts

```diff
@@ -18,6 +18,7 @@ const meta: WsComponentMeta<typeof Box> = {
   isInlineOnly: false,
   isListed: true,
   label: "Box",
+  initialVisibleProps: { tag: "div" },
```

Comment:
still thinking about a better name, maybe `showProps` or `initialProps`?

Reply:
The initial value should be part of the component defaults. Here we are just setting the flag to show the prop control

Reply:
could be: `initialProps: ['tag']`